<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tip</class>
 <widget class="QDialog" name="Tip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>系统提示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Menu.qrc">
    <normaloff>:/icon/Resources/无涯.png</normaloff>:/icon/Resources/无涯.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/other/Resources/背景.png);</string>
  </property>
  <widget class="QLabel" name="lblText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>461</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAC">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRE">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Menu.qrc"/>
 </resources>
 <connections/>
</ui>
